sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContentAsync.java

```diff
@@ -0,0 +1,108 @@
+import java.nio.file.Files;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static com.azure.ai.formrecognizer.implementation.Utility.toFluxByteBuffer;
+
+/**
+ * Async sample for recognizing content information from a document given through a file.
+ */
+public class RecognizeContentAsync {
+    /**
+     * Main method to invoke this demo.
+     *
+     * @param args Unused. Arguments to the program.
+     * @throws IOException Exception thrown when there is an error in reading all the bytes from the File.
+     */
+    public static void main(final String[] args) throws IOException {
+        // Instantiate a client that will be used to call the service.
+        FormRecognizerAsyncClient client = new FormRecognizerClientBuilder()
+                                               .credential(new AzureKeyCredential("{key}"))
+                                               .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+                                               .buildAsyncClient();
+
+        File sourceFile = new File("../formrecognizer/azure-ai-formrecognizer/src/samples/resources/"
+                                       + "sample-forms/forms/Form_1.jpg");
+        byte[] fileContent = Files.readAllBytes(sourceFile.toPath());
+        InputStream targetStream = new ByteArrayInputStream(fileContent);
+
+        PollerFlux<FormRecognizerOperationResult, List<FormPage>> recognizeContentPoller =
+            client.beginRecognizeContent(toFluxByteBuffer(targetStream), sourceFile.length());
+
+        Mono<List<FormPage>> contentPageResultsMono =
+            recognizeContentPoller
+                .last()
+                .flatMap(pollResponse -> {
+                    if (LongRunningOperationStatus.SUCCESSFULLY_COMPLETED.equals(pollResponse.getStatus())) {
+                        System.out.println("Polling completed successfully");
+                        // training completed successfully, retrieving final result.
+                        return pollResponse.getFinalResult();
+                    } else {
+                        return Mono.error(
+                            new RuntimeException(
+                                "Polling completed unsuccessfully with status:" + pollResponse.getStatus()));
+                    }
+                });
+
+        contentPageResultsMono.subscribe(contentPageResults -> {
+            for (int i = 0; i < contentPageResults.size(); i++) {
+                final FormPage formPage = contentPageResults.get(i);
+                System.out.printf("---- Recognized content info for page %d ----%n", i);
+
+                System.out.printf("Page has width: %.2f and height: %.2f, measured with unit: %s%n",
+                    formPage.getWidth(),
+                    formPage.getHeight(),
+                    formPage.getUnit());
+
+                // Table information
+                final List<FormTable> tables = formPage.getTables();
```

Comment:
in .NET we are showing, tables, lines, and selection marks. In case you will like to expand the input, see https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md